### PR TITLE
add submit event

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-1290

### Change description ###

add submit event. 
and testing it with payment test it has already idam , s2s tokens.
I will remove system.out' s

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java [hunk 1]
```diff
@@ -30,6 +39,97 @@
...
     }
 
+    public void startRejectEventAndSubmit(
+        String accessToken,
+        String s2sToken,
+        String userId,
+        String ccdId,
+        Container container
+    ) {
+        Map<String, Object> caseData = getCaseData(accessToken, s2sToken, ccdId);
```
Comment (mdayican) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java:49]: not sure I can use this in submit?

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/PaymentsTest.java [hunk 2]
```diff
@@ -42,6 +42,11 @@ public void should_create_exception_record_with_payments_when_envelope_contains_
         final String s2sToken = s2SClient.getS2SToken();
 
         Await.paymentsProcessed(ccdClient, idamToken, s2sToken, ccdId);
+
+        final String userId = idamClient.getUserId(idamToken);
+
+        System.out.println("userId " + userId);
```
Comment (nitinprabhu) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/PaymentsTest.java:48]: any reason you need this ?

Comment (mdayican) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/PaymentsTest.java:48]: just logging of now

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java [hunk 3]
```diff
@@ -30,6 +39,97 @@
...
+
+        System.out.println("startEventResponse token  " + startEventResponse.getToken());
+
+        CaseDataContent newCaseDataContent = CaseDataContent.builder()
+            .eventToken(startEventResponse.getToken())
+            .event(Event.builder()
+                .id(EVENT_TYPE_ID)
+                .summary(EVENT_SUMMARY)
+                .build())
+            .data(caseData)
```
Comment (nitinprabhu) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java:68]: caseData can be taken from start event response

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java (hunk 1 above)
Comment (nitinprabhu) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java:49]: No need to fetch data just use the case data from start event

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java [hunk 4]
```diff
@@ -3,15 +3,24 @@
...
 import java.util.Map;
 
 import static uk.gov.hmcts.reform.bulkscan.endtoendtests.config.TestConfig.CCD_API_URL;
 
 public class CcdClient {
 
     private static final String BEARER_TOKEN_PREFIX = "Bearer ";
+    public static final String CASE_TYPE = "ExceptionRecord";
+    private static final String EVENT_TYPE_ID = "rejectRecord";
+    private static final String EVENT_SUMMARY = "Reject test an exception record";
```
Comment (rkondratowicz) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java:23]: something wrong in that sentence

Comment (rkondratowicz) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java:23]: generic constant names for very specific values

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/IdamClient.java [hunk 5]
```diff
@@ -15,6 +15,8 @@
 
 public class IdamClient {
 
+    private static final String BEARER_TOKEN = "Bearer ";
```
Comment (rkondratowicz) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/IdamClient.java:18]: this is not a bearer token

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java [hunk 6]
```diff
@@ -30,6 +39,97 @@
         return caseResponse.getData();
     }
 
+    public void startRejectEventAndSubmit(
+        String accessToken,
```
Comment (rkondratowicz) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java:43]: maybe `idamToken` would be clearer

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java [hunk 7]
```diff
@@ -30,6 +39,97 @@
...
+        String s2sToken,
+        String userId,
+        String ccdId,
```
Comment (rkondratowicz) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java:46]: maybe `caseId` would be cleaerer?

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java [hunk 8]
```diff
@@ -30,6 +39,97 @@
         return caseResponse.getData();
     }
 
+    public void startRejectEventAndSubmit(
```
Comment (rkondratowicz) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java:42]: how about simply `rejectCase`?

Comment (mdayican) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java:42]: this method will move from here.

**Pull request merged**